### ISSUE TITLE
Training in  multy-gpu

### ISSUE THREAD

**Issue by a13483685** [AlexeyAB/darknet#5492]
my device have 4 2080ti Gpus ,but when I set the subdivisions=32 in yolov4.cfg and training .shown CUDA Error:out of memory:Resource temoprarily unavailiable in terminal.                                                             And then ,I changed another device(1 GPU 1080ti),also have that question.     How can I solution the problem? thank you very much!

**Comment by WongKinYiu** [comment]
for width=height=512, you can train with batch=64 and subdivisions=32 on 1080ti.
for width=height=416, you can train with batch=64 and subdivisions=16 on 1080ti.

**Comment by AlexeyAB** [comment]
@WongKinYiu 

> for width=height=512, you can train with batch=64 and subdivisions=32 on 1080ti.

I think preferably to use this way (since we use random=1), isn't it?

**Comment by WongKinYiu** [comment]
@AlexeyAB 

yes, and my multi-gpu training on 4 2080ti will finish in this week, I will cheak if the hyper-parameter should be modify or not.
